src/jit/bitset.h

```diff
@@ -279,7 +279,7 @@ class BitSetOps
     class Iter {
       public:
         Iter(Env env, BitSetValueArgType bs) {}
-        bool NextElem(unsigned* pElem) { return false; }
+        int NextElem() { return -1; }
     };
```

Comment:
Fixed, but maybe there is a better way to declare it, rather than `static const unsigned Done = (unsigned)-1;` and use with `BlockSetOps::Iter::Done;`